engine/trends.go

```diff
@@ -39,8 +39,9 @@ func NewTrendS(dm *DataManager,
 		connMgr:     connMgr,
 		filterS:     filterS,
 		cgrcfg:      cgrcfg,
+		crn:         cron.New(),
 		loopStopped: make(chan struct{}),
-		crnTQsMux:   new(sync.RWMutex),
+		crnTQsMux:   &sync.RWMutex{},
```

Comment:
Please keep new to init here, as originally written. Rule out of thumb in our code is to use new for instantiating an empty object and struct literal whenever we are populating also  parameters inside.